"sbt dotr" does not automatically put scala-library on the classpath anymore

This was recently broken (I guess by https://github.com/lampepfl/dotty/pull/3436):
```scala
> dotc tests/pos/HelloWorld.scala
> dotr HelloWorld
Exception in thread "main" java.lang.NoClassDefFoundError: scala/Predef$
        at HelloWorld$.main(HelloWorld.scala:2)
        at HelloWorld.main(HelloWorld.scala)
Caused by: java.lang.ClassNotFoundException: scala.Predef$
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:335)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 2 more
```

Workaround: doing `dotr -classpath /path/to/scala-library-2.12.4.jar` or using the `./bin/dotr` script.

--- a/project/Build.scala
+++ b/project/Build.scala
@@ -635,7 +635,7 @@ object Build {
   def insertClasspathInArgs(args: List[String], cp: String): List[String] = {
     val (beforeCp, fromCp) = args.span(_ != "-classpath")
     val classpath = fromCp.drop(1).headOption.fold(cp)(_ + ":" + cp)
-    beforeCp ::: "-classpath" :: classpath :: fromCp.drop(2)
+    "-classpath" :: classpath :: beforeCp ::: fromCp.drop(2)
   }
 
   lazy val nonBootstrapedDottyCompilerSettings = commonDottyCompilerSettings ++ Seq(

--- a/project/scripts/sbtTests
+++ b/project/scripts/sbtTests
@@ -7,6 +7,7 @@
 echo "testing sbt dotc and dotr"
 mkdir out/scriptedtest0
 ./project/scripts/sbt ";dotc tests/pos/sbtDotrTest.scala -d out/scriptedtest0 ;dotr -classpath out/scriptedtest0 dotrtest.Test" > sbtdotr1.out
+cat sbtdotr1.out
 if grep -e "dotr test ok" sbtdotr1.out; then
     echo "output ok"
 else
@@ -18,7 +19,17 @@ echo "testing sbt dotc -tasty and dotr -classpath"
 mkdir out/scriptedtest1
 mkdir out/scriptedtest2
 ./project/scripts/sbt ";dotc tests/pos/sbtDotrTest.scala -d out/scriptedtest1/; dotc -tasty -classpath out/scriptedtest1/ -d out/scriptedtest2/ dotrtest.Test; dotr -classpath out/scriptedtest2/ dotrtest.Test" > sbtdotr2.out
+cat sbtdotr2.out
 if grep -e "dotr test ok" sbtdotr2.out; then
+    echo "output ok"
+else
+    exit -1
+fi
+echo "testing sbt dotr with no -classpath"
+
+./project/scripts/sbt ";dotc tests/pos/sbtDotrTest.scala; dotr dotrtest.Test" > sbtdotp1.out
+cat sbtdotp1.out
+if grep -e "dotr test ok" sbtdotp1.out; then
     echo "output ok"
 else
     exit -1

